### DISCUSSION TITLE
TTS大声朗读接口貌似不能用了

### DISCUSSION THREAD

**Discussion by limne** [jing332/tts-server-android#225] [General]
今天早上tts大声朗读接口调用403，貌似不能用了，大佬能不能看看是微软url变了，还是非edge调用用不了？

**Comment by h2dcc** [comment]
对啊，连接不上，得科学上网

**Reply by huangsijun17** [reply]
我现在挂了梯子，似乎也不能用了。
我看了一下bing翻译的朗读接口，似乎接口变了？

**Reply by limne** [reply]
应该是接口变了，我看edge浏览器还可以用的

**Reply by sunbodyyy** [reply]
这个软件作者是不是不更新了

**Reply by zxssaa** [reply]
看显示的2个月前还有人提交修改代码，但是没有更新安装包。

**Reply by huangsijun17** [reply]
Action中下载编译版本？我不太想为这个装个安卓SDK来编译APK。

**Reply by h2dcc** [reply]
在QQ频道找到另一个“微软翻译”插件可以使用。下载密码 ozRq  https://www.123pan.com/s/qMUjVv-qSZPv.html

**Reply by toiilf** [reply]
@h2dcc “微软翻译” 插件咋用的，求大佬指导

**Reply by h2dcc** [reply]
> @h2dcc “微软翻译” 插件咋用的，求大佬指导

下载到手机后选择用tts app打开，然后选插件导入。或者app里添加插件选json文件就行了。
之后app添加插件tts。

**Reply by toiilf** [reply]
> > @h2dcc “微软翻译” 插件咋用的，求大佬指导
> 
> 下载到手机后选择用tts app打开，然后选插件导入。或者app里添加插件选json文件就行了。 之后app添加插件tts。

搞定了，感谢大佬

**Reply by 550-q** [reply]
> 看显示的2个月前还有人提交修改代码，但是没有更新安装包。

2个月前的更新有挂不住后台的严重bug，目前可用的最新版本是1.24.062213，新于这个版本的别用。
现在接口也炸了，只能挂vpn或用插件。

**Reply by h2dcc** [reply]
> > 看显示的2个月前还有人提交修改代码，但是没有更新安装包。
> 
> 2个月前的更新有挂不住后台的严重bug，目前可用的最新版本是1.24.062213，新于这个版本的别用。
> 现在接口也炸了，只能挂vpn或用插件。

果然，看了下首页介绍居然都更新在蓝奏云上。。我还是用的release里边去年版本

**Reply by chenhuxiang** [reply]
现在的意思是 不用梯子是用不了了吗？可是我用了梯子也会报错

**Reply by 550-q** [reply]
> 现在的意思是 不用梯子是用不了了吗？可是我用了梯子也会报错

我刚试了，用vpn可用，可能你的vpn有问题。

**Reply by xingkong2100** [reply]
挂VPN可用，应该是接口被墙了，谁有解决方法，挂VPN手机太费电了

**Reply by 550-q** [reply]
> 挂VPN可用，应该是接口被墙了，谁有解决方法，挂VPN手机太费电了

有插件可用，楼上有人分享

**Reply by bz199707** [reply]
等更新。

**Reply by huangsijun17** [reply]
> ozRq

确认了一下，手机里的微软规则组自动直连了。家里路由器没配规则。

**Reply by zqy1-1** [reply]
> > @h2dcc “微软翻译” 插件咋用的，求大佬指导
> 
> 下载到手机后选择用tts app打开，然后选插件导入。或者app里添加插件选json文件就行了。 之后app添加插件tts。

谢谢，亲测可用

**Reply by linweifu** [reply]
> > @h2dcc “微软翻译” 插件咋用的，求大佬指导
> 
> 下载到手机后选择用tts app打开，然后选插件导入。或者app里添加插件选json文件就行了。
> 之后app添加插件tts。

插件可以用了，感谢

**Reply by wjywjwww** [reply]
的确可以用，还挺好用

**Reply by QinghuanWei** [reply]
> > > @h2dcc “微软翻译” 插件咋用的，求大佬指导
> > 
> > 
> > 下载到手机后选择用tts app打开，然后选插件导入。或者app里添加插件选json文件就行了。
> > 之后app添加插件tts。
> 
> 插件可以用了，感谢

![Screenshot_20241101_004708](https://github.com/user-attachments/assets/f0c6c474-f3c6-4896-b496-c3f0da2299a4)
咋用呀，还是不行

**Reply by asgsdbrseg** [reply]
插件地址分享一下

**Reply by asgsdbrseg** [reply]
> 在QQ频道找到另一个“微软翻译”插件可以使用。下载密码 ozRq https://www.123pan.com/s/qMUjVv-qSZPv.html

链接过期了，麻烦补链

**Reply by h2dcc** [reply]
> > 在QQ频道找到另一个“微软翻译”插件可以使用。下载密码 ozRq https://www.123pan.com/s/qMUjVv-qSZPv.html
> 
> 链接过期了，麻烦补链

建议换multiTTS，这个即便换插件也卡。要么就原来的挂外网用。

**Reply by h2dcc** [reply]
> 插件地址分享一下

https://github.com/h2dcc/codespaces-blank/blob/main/MS-TRANS.ini 
这网盘我也没用过，刚找到文件传上github了，把这个另存json就行。

**Reply by H945435643** [reply]
> > 插件地址分享一下
> 
> https://github.com/h2dcc/codespaces-blank/blob/main/MS-TRANS.ini 这网盘我也没用过，刚找到文件传上github了，把这个另存json就行。

没了

**Reply by Allone** [reply]
> > > 在QQ频道找到另一个“微软翻译”插件可以使用。下载密码 ozRq https://www.123pan.com/s/qMUjVv-qSZPv.html
> > 
> > 链接过期了，麻烦补链
> 
> 建议换multiTTS，这个即便换插件也卡。要么就原来的挂外网用。


multiTTS很不错。

**Comment by linweifu** [comment]
是的，今天凌晨突然用不了

**Comment by luxiaoge01** [comment]
不能用啦

**Comment by gexgd0419** [comment]
Edge语音对来自国内IP地址的请求增加了对 `Sec-MS-GEC` 和 `Sec-MS-GEC-Version` 参数的验证。

edge-tts 已经更新了解决方案，参数目前的算法见这里：https://github.com/rany2/edge-tts/issues/290#issuecomment-2464956570

**Comment by huoyuanpang** [comment]
能给tts server朗读增加预缓存功能么，阅读当前段落预缓存第二段落，分割长句打开的可以多缓存几个段落

**Reply by huangsijun17** [reply]
请单开需求，这是个BUG的反馈。

**Comment by jing332** [comment]
Edge已使用插件代替：https://github.com/jing332/tts-server-android/issues/239#issuecomment-2671220214
